<launch>
	<arg name="rviz" default="true" />
	<node pkg="generalp" type="findInterestSure" name="findInterestSure" output="screen">
        <param name="featureSize" value="8" />
        <param name="featureSamplingRate" value="15" />
        <param name="normalsScale" value="3.93091" />
        <param name="normalSamplingRate" value="5" />
        <param name="minimumCornerness" value="0.00000869" />
        <param name="inputCloud" type="string" value="/velodyne_cloud_2" />
        <param name="outputCloud" type="string" value="/velodyne_cloud_3" />
	</node>
	<node pkg="velodyne_pointcloud" type="cloud_node" name="cloud_node">
		<param name="calibration" value="$(find anestis_loam_mod)/config/vlp16.yaml"/>
		<param name="min_range" value="0.4"/>
		<param name="max_range" value="135.0"/>
	</node>
	<!--node pkg="generalp" type="scanFix" name="scanFix"/-->

<!--GOOOOOD/-->

  <group if="$(arg rviz)">
    <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find generalp)/loam_velodyne.rviz" />
  </group>
  <node pkg="tf" type="static_transform_publisher" name="static_transform_publisher3" args="0 0 0 0 0 1.5  world camera_init 50"/>
<param name="use_sim_time" value="true" />

</launch>
